### PR TITLE
Route53 zone & records

### PR DESCRIPTION
**Pull request by JohnNMwangi**
A route53 zone with an option to create multiple records.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,75 @@
+resource "aws_route53_zone" "this" {
+  name              = var.name
+  comment           = "Route53 hosted zone"
+  delegation_set_id = var.vpc == null ? var.delegation_set_id : null
+  force_destroy     = var.force_destroy
+  dynamic "vpc" {
+    for_each = var.vpc
+    content {
+      vpc_id = vpc.value.vpc_id
+      #vpc_region = lookup(vpc.value, "vpc_region", null)
```
Comment (patrickmukumbu) [inline main.tf:10]: Just curious why this line is commented out

**Review by patrickmukumbu — COMMENTED** [review]
`terraform.tf` is empty. See my other comment

File: main.tf (hunk 1 above)
Comment (JohnNMwangi) [reply main.tf]: That was part of troubleshooting haha, shouldn't be commented out

**Review by patrickmukumbu — APPROVED** [review]
Looks Great!

**Pull request merged**